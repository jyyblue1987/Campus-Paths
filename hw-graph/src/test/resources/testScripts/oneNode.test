# To test if we can add a node to the graph

# Create a graph
CreateGraph A

# Add a node
AddNode A n

# Print out the nodes in the graph
ListNodes A
ListChildren A n