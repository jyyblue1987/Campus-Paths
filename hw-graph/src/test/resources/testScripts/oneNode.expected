# To test if we can add a node to the graph

# Create a graph
created graph A

# Add a node
added node n to A

# Print out the nodes in the graph
A contains: n
the children of n in A are: